findimports.py

```diff
@@ -97,7 +99,7 @@
 from operator import attrgetter
 
 
-__version__ = '2.3.1.dev0'
+__version__ = '2.3.0'
```

Comment:
Please do not touch `__version__` .  As part of the release process I remove the `.dev0` suffix and check for the need of semver-ish version number increases by looking at the changelog (in CHANGES.rst).

I used to preemptively increase the next upcoming version (like '2.3.1.dev0' -> '2.4.0.dev0') when I implemented a new feature, but this way it's easy to accidentally bump the minor version twice, so I mostly stopped.

Reply:
That was accidental. I did work in site packages, then copied over. my bad